content/en/docs/install-operate/deployment.md

```diff
@@ -1,13 +1,23 @@
 ---
 title: Deployment
-description: Installing Falco on a Cluster
+description: Installing Falco on a Kubernetes Cluster
 aliases: [/docs/getting-started/deployment]
-weight: 40
+weight: 45
 ---
 
+The guides - [Download](/docs/install-operate/download/), [Install](/docs/install-operate/installation/), [Running](/docs/install-operate/running/) - focused on the systemd service deployment method, detailing how to primarily install and run Falco directly on the host. Additionally, these guides explored the alternative options of installing and running Falco within a container, with a driver installed on the underlying host.
+
+The container deployment option is particularly enticing for Kubernetes settings, which will be discussed in more detail now. 
+
+{{% pageinfo color="info" %}}
+
+Falco only needs to run once per Linux operating system, irrespective of the deployment strategy, whether on a bare-metal server or a virtual machine (VM). It takes advantage of the shared kernel in containerized infrastructures like Kubernetes through the installation of a kernel driver on the underlying host. Thus, deploying Falco within a container with the driver installed on the host is equivalent to deploying it directly on the Linux host. This allows seamless monitoring of both host and container workloads *without* the need for sidecars.
```

Comment:
This is a very very common misconception and point of confusion @nate-double-u if you have even more ideas on how to highlight this would appreciate it!

Reply:
So installing the drivers on the host is key -- but you still need to install Falco within each container you want monitored?

Or, does this mean that if you install Falco on the bare-metal server, you get monitoring on all the containerized infra "for free."

Reply:
Glad I asked so it's not yet clear.


> Or, does this mean that if you install Falco on the bare-metal server, you get monitoring on all the containerized infra "for free."

It's the second one.

And often people also think you can ONLY run Falco in Kubernetes and not on a regular host or VM. 
And it does not have to be the bare-metal server, e.g. in EKS, GKS, AKS in fact the "host" is a VM ...

Any text suggestions to clearly convey all these? The current version is not there yet ...

Reply:
> So installing the drivers on the host is key 

It kind of is the only way it can work. And host for most people as mentioned above in EKS, GKS, AKS is a VM